.github/workflows/tests.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install setuptools if Python 3.12
+        if: startsWith(matrix.python-version, '3.12')
+        run: python -m pip install --upgrade pip setuptools
+

```

Comment:
Let's remove this and install setuptools unconditionally in the "Install plugin" step